src/main/java/Event.java

```diff
@@ -1,16 +1,20 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 public class Event extends Task {
```

Comment:
This public class could use a java doc too

Reply:
Done!